<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Определение средней интенсивности цвета</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(220, 220, 220);
color: black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(205,205,205);
border-radius: 10px;
</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QFrame" name="frame">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Sans Serif&quot;;
border-radius:10px;
background: rgb(205, 205, 205);
</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="loadImageButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Sans Serif</family>
             <pointsize>9</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>false</underline>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 9pt &quot;Sans Serif&quot;;
background: rgb(220, 220, 220);
border: 2px solid rgb(194, 194, 194);
border-radius: 5px;
padding: 3px;

</string>
           </property>
           <property name="text">
            <string>Загрузить изображение</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="tools">
           <item>
            <layout class="QGridLayout" name="zoomLayout">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item row="1" column="0">
              <widget class="QToolButton" name="unZoomButton">
               <property name="minimumSize">
                <size>
                 <width>32</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">padding: 5px;
background:rgb(220,220,220);
border:2px solid rgb(185,185,185);
border-radius: 4px;</string>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QToolButton" name="zoomButton">
               <property name="minimumSize">
                <size>
                 <width>32</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background:rgb(220,220,220);
border:2px solid rgb(185,185,185);
border-radius: 4px;
</string>
               </property>
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="zoomLabelUP">
               <property name="font">
                <font>
                 <family>Sans Serif</family>
                 <pointsize>8</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">font-size:8pt;</string>
               </property>
               <property name="text">
                <string>Увеличить масштаб</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="zoomLabelOut">
               <property name="font">
                <font>
                 <family>Sans Serif</family>
                 <pointsize>8</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">font-size:8pt;</string>
               </property>
               <property name="text">
                <string>Уменьшить масштаб</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Инструменты:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="mainInstruments">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="rulerButton">
               <property name="styleSheet">
                <string notr="true">background:rgb(220,220,220);
border:2px solid rgb(185,185,185);
border-radius: 4px;</string>
               </property>
               <property name="text">
                <string>Линейка</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="rectButton">
               <property name="styleSheet">
                <string notr="true">background:rgb(220,220,220);
border:2px solid rgb(185,185,185);
border-radius: 4px;</string>
               </property>
               <property name="text">
                <string>Прямоугольник</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWidget" name="RectIntens" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="minLabel">
              <property name="text">
               <string>Мин. инт. : 0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="maxLabel">
              <property name="text">
               <string>Макс. инт. : 0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="avgLabel">
              <property name="text">
               <string>Средн. инт. : 0</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="allPngIntens" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="minAllLabel">
              <property name="text">
               <string>П макс. инт. : 0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="maxAllLabel">
              <property name="text">
               <string>П мин. инт. : 0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="avgAllLabel">
              <property name="text">
               <string>П сред. инт. : 0</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="logTextEd">
           <property name="maximumSize">
            <size>
             <width>175</width>
             <height>150</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:1px solid black;
border-radius:0;
background:white;
color:black;
font: 6pt &quot;Sans Serif&quot;;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-bottom:2px solid rgb(205,205,205);
font: 10pt &quot;Sans Serif&quot;;
</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="loadImageMenu"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="loadImageButton_2">
   <property name="text">
    <string>Загрузить изображение</string>
   </property>
  </action>
  <action name="loadImageMenu">
   <property name="text">
    <string>Загрузить изображение</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
